tools/README.md

```diff
@@ -0,0 +1,39 @@
+# Tools for Kiali maintainers
+
+This folder contains tools for maintainers of the Kiali project.
+
+## Mock graph data
+
+There's two cli tools to help with testing large graph topologies without needing to deploy the underlying resources.
+
+### generator
+
+The generator creates a json file with a mock `/api/namespaces/graph` response.
+
+Running the following command will create a json file with 50 apps (workloads + services). There is some randomness in the graph generation so running the command multiple times with the same options will yield different results.
```

Comment:
I did this only to read below that I didn't need to do it :-)

Reply:
Whoops yeah I'll include that information further up